frontend/src/services/graphql/Vpc/getNetwork.js

```diff
@@ -1,22 +0,0 @@
-import { gql } from 'apollo-boost';
-
-export const getNetwork = (vpcUri) => ({
-  variables: {
-    vpcUri
-  },
-  query: gql`
-    query getNetwork($vpcUri: String!) {
```

Comment:
Same here. We have this `getNetwork` API  in the backend only to serve this query